app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -225,6 +228,16 @@ class NavigationOverlayFragment : Fragment() {
         compositeDisposable.clear()
     }
 
+    fun dispatchKeyEvent(event: KeyEvent): Boolean {
+        // This broke, which went unnoticed for several releases, when it was refactored into a different Fragment:
```

Comment:
"This broke" can you add more context like "MenuInteractionMonitor broke" or "Menu telemetry broke"